hardware/asic/umc130/memory/Makefile

```diff
@@ -1,17 +1,18 @@
-FIRM_DIR:=../../../../software/firmware
-BOOT_DIR:=../../../../software/bootloader
+ROOT_DIR:=../../../..
+include $(ROOT_DIR)/system.mk
 
-BAUD:=115200
-FREQ:=100000000
+# in the future, BAUD and FREQ is to be defined by system.mk; this .mk file still needs to be changed for this end
+BAUD_TMP:=115200
```

Comment:
future is now, this is already defined, use it

Reply:
BAUD and FREQ ar probably not used in the hardware. Try deleting them. As far as I remember only software and testbench needs these vars that are used to init the UART in run time.

Reply:
BAUD and FREQ deleted